src/main/java/org/openstreetmap/atlas/generator/PbfLocator.java

```diff
@@ -32,15 +36,13 @@
  *
  * @author matthieun
  */
-public class PbfLocator implements Serializable
```

Comment:
Why did we ditch `Serializable`?

Reply:
Yes. I realized Sonar complained that some of its members were not serializable, and not transient, which means we never used it, otherwise it would have failed.

Reply:
good catch!